public/app/plugins/datasource/prometheus/components/PromQueryEditor.tsx

```diff
@@ -24,11 +24,29 @@ const INTERVAL_FACTOR_OPTIONS: Array<SelectableValue<number>> = map([1, 2, 3, 4,
   label: '1/' + value,
 }));
 
+const DEFAULT_STEP_OPTION: SelectableValue<StepType> = {
+  value: 'min',
+  label: 'Minimum',
+};
+
+const STEP_OPTIONS: Array<SelectableValue<StepType>> = [
+  DEFAULT_STEP_OPTION,
+  {
+    value: 'max',
+    label: 'Maximum',
+  },
+  {
+    value: 'exact',
+    label: 'Exact',
+  },
+];
+
```

Comment:
@ivanahuckova is this what you had in mind with the default step option?

Reply:
Yes